### PR TITLE
Make rake to development dependency

### PR DESCRIPTION
**Pull request by giginet**
I integrated `danger-swiftlint` to my project.
Then rake version is reverted to older version.

I think `rake` should be `development_dependency`.

### CODE REVIEW

File: .ruby-version [hunk 1]
```diff
@@ -1 +1 @@
-2.3.0
+2.3
```
Comment (giginet) [inline .ruby-version]: Ruby 2.3.4 have been released

**Comment by giginet** [comment]
I read this commit log.

https://github.com/giginet/danger-swiftlint/commit/7fd8d28886a7cd5f26043c43776841669453fd43

However, I can't understand why this change is required.

**Comment by ashfurrow** [comment]
Gotcha, thanks for the pull request. @mokagio do you remember why rake was moved as a regular dependency?

**Comment by thiagofelix** [comment]
Is it because rake is used [during install](https://github.com/ashfurrow/danger-swiftlint/blob/master/danger-swiftlint.gemspec#L19) to download the swiftlint extension?

Maybe if it is on dev_dependencies it might not be available during install therefore the installation will fail.

I am checking if this is true in regards to the gem/bundler behaviour.

**Comment by giginet** [comment]
I understood.

I made rake version more openly.

**Comment by mokagio** [comment]
Hi folks, sorry for the delay in replying to this.

I did not remember why I added rake as an install dependency, but luckily [git did](https://github.com/ashfurrow/danger-swiftlint/commit/7fd8d28886a7cd5f26043c43776841669453fd43):

> This is so that installation in vendored setup, where the gems are installed locally to the repository rather than globally in the user's path (sort of), won't fail in case Rake is not installed for some reason.

Give this, I think that the current dependency definition is 👌 👌 👌 

```
spec.add_dependency 'rake', '> 10'
```

File: .ruby-version [hunk 2]
```diff
@@ -1 +1 @@
-2.3.0
+2.3.4
```
Comment (mokagio) [inline .ruby-version]: Question: would this impact the gem compilation or just the development tooling?

Comment (giginet) [inline .ruby-version]: I have no idea.
As I checked, there are no problems.
All specs are still passed.

If it causes any problem,`.ruby-version` never cause-effect for gem users. So I think it couldn't be big problem.

Should I revert this changes?

Comment (mokagio) [inline .ruby-version]: Nope no need 😄 It was just my curiosity 👍 

My guess is that it won't affect the end users of the gem too 😉

**Comment by giginet** [comment]
Could you release 0.5.1?

**Comment by ashfurrow** [comment]
Sure thing, will do this morning.

**Comment by ashfurrow** [comment]
Okay, should be all set. Thanks again!

File: .ruby-version (hunk 2 above)
Comment (thiagofelix) [inline .ruby-version]: I am quite certain that this file is not used during compilation, It is supposed to mean more like a hint for ruby managers such as rvm and rbenv =)
